### ISSUE TITLE
Make IQ/Tuning Post Slow Tune

### ISSUE THREAD

**Issue by msilvafe** [slaclab/pysmurf#65]
We should be able to generate resonator sweeps of magnitude, phase, and IQ circles like is output in setup notches of only the channels that are active after you have killed whichever channels you don't want to be using.

**Comment by ruck314** [comment]
This request looks functionally identical to #171 ("New Eta Plots w/o having to run Setup_Notches again"), which was filed later with more specific implementation suggestions:

> ...rewriting the `plot_tune_summary` function to include an option for running full eta_scans on only the channels that are on (not all of the channels in find_freq) and producing new eta plots from these eta_scans.

Open PR #955 (branch `issues-171`, base `main`) implements exactly that, and the same code path satisfies what is asked here in #65:

- New method `eta_scan_on_channels(band, ...)` in `python/pysmurf/client/tune/smurf_tune.py`. It reuses the firmware-driven serial sweep mechanism `setup_notches` already uses (`set_eta_scan_freq` + `set_run_serial_find_freq`), but only populates the eta-scan freq register for the channels currently on, i.e. `which_on(band)` — which is exactly "the channels that are active after you have killed whichever channels you don't want to be using" from this issue. Each on-channel is centered on its current `channel_to_freq`, so any drift since `setup_notches` is captured. Refreshes `freq_eta_scan` / `resp_eta_scan` / `eta` / `eta_mag` / `eta_phase` in `self.freq_resp[band]['resonances']`.
- `plot_tune_summary` gains `rerun_eta_scans`, `sweep_width`, `df_sweep` kwargs. With `rerun_eta_scans=True` it calls `eta_scan_on_channels` first and then restricts the per-channel eta plots to those on-channels. The existing `plot_eta_fit` already produces the magnitude, phase, and IQ-circle subplots this issue asks for.

Typical workflow once PR #955 lands:

```python
S.find_freq(band)
S.setup_notches(band, new_master_assignment=True)
S.run_serial_gradient_descent(band)
S.run_serial_eta_scan(band)
# ... kill channels you don't want, e.g. S.set_amplitude_scale_channel(band, ch, 0) ...
S.plot_tune_summary(band, rerun_eta_scans=True, plot_summary=False)
```

That last call sweeps and produces magnitude/phase/IQ-circle plots for only the surviving on-channels — no `setup_notches` rerun, no full-band sweep.

Recommend closing this as a duplicate of #171 (resolved by #955), and dropping the `wontfix` label since the request is being fulfilled.
